docs/Using Fleet/MDM-macOS-setup.md

```diff
@@ -95,7 +204,23 @@ In the output you should see that package has a "signed" status.
 
 ### Step 3: upload the package to Fleet
 
-1. Upload the package to a storage location (ex. S3 or GitHub). During step 4, Fleet will retrieve the package from this storage location and host it for deloyment.
+In Fleet, you can upload the package using the Fleet UI or fleetctl command-line tool.
+
+The Fleet UI method is a good start if you're just getting familiar with Fleet.
```

Comment:
Don't thik we need to exlain this here.

Reply:
Sounds good. I think cutting is makes sense